README.md

```diff
@@ -61,231 +24,141 @@ For more information on the importance of a professional README for your reposit
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Rails</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://expressjs.com/">Rails</a></li>
   </ul>
 </details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- **[Ruby]**
+- **[Rails]**
+- **[postgtreSQL]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+Git clone https://github.com/SAZZAD1Q2/BudgetMate.git
+Run in your browser
 
 ### Prerequisites
 
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+To run this project you need:
+Git
+An IDE like vs code . bracket etc.
+Code Editor
+Browser
 
 ### Setup
 
-Clone this repository to your desired folder:
+Clone this repository to your desired folder: https://github.com/SAZZAD1Q2/BudgetMate.git
+Opent it to your code editor
+open it then to your browser.
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
 
 ### Install
 
 Install this project with:
+Github or git or vs code. For this open your terminal and proceed some steps:
+a. Install node package using the command : npm install
+b. Then run the website using the command : npm start
+c. Test the code using jest snapShots and for this use the Command : npx jest
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Deployment
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
 
-You can deploy this project using:
+- [Live Demo Link](https://mobile-app-zn.onrender.com)
 
-<!--
-Example:
+<p align="right">(<a href="#readme-top">back to top</a>)</p
 
-```sh
+<!-- Presentation Video -->
+## 🎥 Presentation Video <a name="presentation-video"></a>
 
-```
- -->
+- [Presentation Video Link](https://www.loom.com/share/590eccee11f94d65b2fb7aaddeaedc70?sid=1d276fce-a84b-4b5e-9db2-02254369e9f5)
```

Comment:
- [ ] In your video presentation you did not show us a part of your code you are proud of, kindly include that